### DISCUSSION TITLE
Pinecone Integration

### DISCUSSION THREAD

**Discussion by drallen-ai** [vercel/community#5854] [Help] [answered]
### Question

I'm trying to add Pinecone using the integration feature in Vercel and its giving me an error... I've tried clearing cache, different browsers, adding to all projects or just special ones and I get the same error... any ideas? It says to "remove integration and try again" but there is nothing to remove as its not in my account. Just to check, I tried adding Replicate using the same process and had no issues with that. I did a Pinecone search and nothing came up that I found. Thanks. 
![pinecone_vercel_error_integration](https://github.com/vercel/community/assets/126411784/a9bd3cc2-d231-4cea-ba03-3fdd3fb9a44e)

**Comment by tdonia** [comment, marked as answer]
Hi @drallen-ai -- this issue should be resolved. please try again and let me know if you encounter any issues.
